packages/sdk-router/README.md

```diff
@@ -46,6 +46,28 @@ const synapseSDK = new SynapseSDK(chainIds, providers)
 
 ## Bridging
 
+### Router Deployments
+
+The Routers for each module are deployed to consistent addresses across all chains, with a few exceptions noted below.
+
+| Bridge Module | Chain   | Address                                      |
+| ------------- | ------- | -------------------------------------------- |
+| SynapseBridge | All[^1] | `0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a` |
+| SynapseBridge | Blast   | `0x0000000000365b1d5B142732CF4d33BcddED21Fc` |
+| SynapseCCTP   | All     | `0xd5a597d6e7ddf373a92C8f477DAAA673b0902F48` |
+| SynapseRFQ    | All     | `0x00cD000000003f7F682BE4813200893d4e690000` |
```

Comment:
are the fastbridge routers all at the same address even though the bridge contract is different on blast and linea?

Reply:
Yes, the FastBridgeRouter has the same address regardless of FastBridge address